File: Iniciante/URI 1037.cs
using System; 
 
class URI {
 
    static void Main(string[] args) { 
        double valor = double.Parse(Console.ReadLine());
        if (valor >= 0 && valor <= 25) {
            Console.WriteLine("Intervalo [0,25]");
        } else if (valor > 25 && valor <= 50) {
            Console.WriteLine("Intervalo (25,50]");
        } else if (valor > 50 && valor <= 75) {
            Console.WriteLine("Intervalo (50,75]");
        } else if (valor > 75 && valor <= 100) {
            Console.WriteLine("Intervalo (75,100]");
        } else {
            Console.WriteLine("Fora de intervalo");
        }
    }
    
}

File: Iniciante/URI 1037.dart
import 'dart:io';

void main() {
  double valor = double.parse(stdin.readLineSync());
  if (valor >= 0 && valor <= 25) {
    print("Intervalo [0,25]");
  } else if (valor > 25 && valor <= 50) {
    print("Intervalo (25,50]");
  } else if (valor > 50 && valor <= 75) {
    print("Intervalo (50,75]");
  } else if (valor > 75 && valor <= 100) {
    print("Intervalo (75,100]");
  } else {
    print("Fora de intervalo");
  }
}
